### PR TITLE
Add FENIX ecosystem

### PR DESCRIPTION
**Pull request by joeblau**
Just watched @mariashen give a talk at BuildETH. Following up 🫡

### CODE REVIEW

File: data/ecosystems/f/fenix.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Ecosystem Level Information
+title = "Fenix Protocol"
```
Comment (eherrerosj) [inline data/ecosystems/f/fenix.toml:2]: hey @joeblau, thanks for the contribution. I see the Fenix is built on Ethereum. If so, please add "Fenix Protocol" to the list of sub_ecosystems on file `ethereum.toml` under `data/ecosystems/e`

**Review by eherrerosj — CHANGES_REQUESTED** [review]
hey @joeblau, thanks for the contribution. I see the Fenix is built on Ethereum. If so, please add "Fenix Protocol" to the list of sub_ecosystems on file ethereum.toml under data/ecosystems/e

**Comment by joeblau** [comment]
FENIX will exist on 13 chains... Ethereum, BSC, Polygon, Avalanche, EthereumPOW, Moonbeam, Evmos, Fantom, DogeChain, OKChain, Arbitrum, Optimism, and PulseChain. Our community views the EVM as the value proposition, Ethereum is just one of the chains we're launching on.

That being said I've updated the PR.

File: data/ecosystems/e/ethereum.toml [hunk 2]
```diff
@@ -6794,6 +6796,9 @@ url = "https://github.com/LaTribuWeb3/dex-history-fetcher"
 [[repo]]
 url = "https://github.com/LaTribuWeb3/fei-protocol-core"
 
+[[repo]]
+url = "https://github.com/atomizexyz/fenix"
```
Comment (eherrerosj) [inline data/ecosystems/e/ethereum.toml:6800]: you can remove this repo, since it's already in the Ethereum's child "Fenix" ecosystem

**Review by eherrerosj — CHANGES_REQUESTED** [review]
We only need the ecosystem name in Ethereum, our scripts do the rest to come up with the entire tree of relations.

Wrt to the multi-chain compatibility. We have this internal concept of "primary" and "supported" ecosystems. We consider a relation between ecos to be primary when it has a currency / gov token in that network or it was originally built there and deployed afterwards to other chains. If Ethereum happen to not be the case of primary to Fenix please lmk

**Review by eherrerosj — APPROVED** [review]
Perfect, merging! Thank you for the contribution

**Pull request merged**